UBA-DE-Emissionsituation/space-emissions
# Sketch workbook implementation of the space emissions tool

Implementing the ideas from the wiki at https://github.com/UBA-DE-Emissionsituation/space-emissions/wiki and from issue #2.
Please feel free to comment and exchange thoughts/ideas.

How to test this:
1. Make sure to use a machine that has Jupyter Notebooks installed (or see hint below)
2. Get a local copy of this repo: `git clone https://github.com/UBA-DE-Emissionsituation/space-emissions.git`
3. Switch to this branch: `git checkout origin/sketch`
4. Open the notebook in the root folder and run all cells

A good online environment to test this actually comes with code-de.org, simply log into `https://jupyter.code-de.org/user/<user name>/lab` - should work great (use full email address as user name)!
